test/fixtures/hazard_categories.yml

```diff
@@ -0,0 +1,7 @@
+# Read about fixtures at http://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
+
+one:
+  name: MyString
+
+two:
+  name: MyString
```

Comment:
🐘Not sure how clean we are trying to keep things this time but I would remove the unused fixtures and test scaffolding that will at file cruft over time.